packages/patternfly-4/react-catalog-view-extension/node-sass-patternfly-importer.js

```diff
@@ -1,5 +1,9 @@
+// eslint-disable-next-line @typescript-eslint/no-var-requires
```

Comment:
Just curious on this. instead of disable the check why not switch it to proper ES6 typescript format?
import path = require('path');

This is in multiple places

Reply:
Good thought - I tried that initially but then it causes prettier to throw an error, and at that point it seemed like multiple fixes to get around a linter issue:
![Screen Shot 2020-02-11 at 5 00 36 PM](https://user-images.githubusercontent.com/8651509/74283691-516a3b80-4cf0-11ea-9b92-54334c51dbb0.png)